extensions/amp-analytics/0.1/crypto-impl.js

```diff
@@ -93,6 +104,20 @@ export class Crypto {
       return result;
     });
   }
+
+  /**
+   * Loads Crypto polyfill library.
+   * @returns {!Promise}
+   * @private
+   */
+  loadPolyfill_() {
+    if (this.polyfillPromise_) {
+      return this.polyfillPromise_;
+    }
+    return this.polyfillPromise_ = extensionsFor(this.win_)
```

Comment:
Need to add the `polyfillPromise_` property in the constructor, regardless of `subtle_`.